src/import/ImportWallet.tsx

```diff
@@ -176,9 +179,6 @@ function ImportWallet({ navigation, route }: Props) {
         <SafeAreaInsetsContext.Consumer>
           {(insets) => (
             <View style={styles.container}>
-              <UseBackToWelcomeScreen
-                backAnalyticsEvents={[OnboardingEvents.restore_account_cancel]}
```

Comment:
Are we still tracking this somewhere else?

Reply:
We are not. I'll add it back maybe on press of the back button on `importWallet.tsx`?

Reply:
why does this component need to be removed in the first place?

Reply:
This component was causing a freeze on import wallet after setting pin during the restore flow.